core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -423,7 +447,10 @@ export class PickerColumnInternal implements ComponentInterface {
           }
           return (
             <button
-              tabindex="-1"
+              role="radio"
```

Comment:
Role `radio` seemed like the most appropriate role based on the ARIA spec (single selection allowed in a group at a time).